st2common/tests/unit/test_db.py

```diff
@@ -51,6 +54,38 @@
 DUMMY_DESCRIPTION = 'Sample Description.'
 
 
+class DbIndexNameTestCase(TestCase):
+    """
+    Test which verifies that model index name are not longer than the specified limit.
+    """
+    LIMIT = 65
```

Comment:
Minor nit-pick. Make `LIMIT` lowercase if used in the class or make it a constant at top of module.  This is more consistent with our coding.